frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -788,9 +861,89 @@ const Chart = (props: ChartProp) => {
                 }
             }
         },
-        [getRealIndex, dispatch, updateVars, propagate, props.onChange, config.traces.length, module]
+        [getRealIndex, dispatch, updateVars, propagate, props.onChange, config.traces.length, module],
     );
 
+    useEffect(() => {
+        if (!dataPl.length || !animationDataValues) return;
```

Comment:
May be we want to clean the 'frame data' at this point?
One way not to animate any longer...